### PR TITLE
Master -> Main Rename

### PR DESCRIPTION
**Pull request by dannylamb**
Part of https://github.com/Islandora/documentation/issues/1595

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ N.B. This virtual machine **should not** be used in production.
 1. [VirtualBox](https://www.virtualbox.org/)
   * Be sure to install a version of VirtualBox that [is compatible with Vagrant](https://www.vagrantup.com/docs/virtualbox/)
 2. [Vagrant](http://www.vagrantup.com)
+  * Important: be sure to install Vagrant version 2.0.3 or higher
+  * If upgrading from a previus version run ```bash vagrant plugin update``` to avoid plugin issues
```
Comment (whikloj) [inline README.md]: Typo "previus"

**Review by whikloj — CHANGES_REQUESTED** [review]

**Comment by DonRichards** [comment]
@dannylamb Ping

File: README.md (hunk 1 above)
Comment (DonRichards) [reply README.md]: @whikloj This should be resolved. Are you good with resolving conversation?

**Review by whikloj — APPROVED** [review]

**Comment by DonRichards** [comment]
@whikloj Thanks for getting this.

**Pull request merged**